behat.yml.dist

```diff
@@ -4,5 +4,51 @@
 # This file is referenced in Sylius-Standard v1.0.0 - v1.3.x
 
 imports:
-    - src/Sylius/Behat/Resources/config/profiles.yml
     - src/Sylius/Behat/Resources/config/suites.yml
+
+default:
+    formatters:
+        pretty:
+            verbose: true
+            paths: false
+            snippets: false
+
+    extensions:
+        Lakion\Behat\MinkDebugExtension:
+            directory: etc/build
+            clean_start: false
+            screenshot: true
+
+        Behat\MinkExtension:
+            files_path: "%paths.base%/src/Sylius/Behat/Resources/fixtures/"
+            base_url: "http://localhost:8080/"
```

Comment:
AFAIK it is not mentioned there, but it's a change not connected to this PR (I've copied it from `src/Sylius/Behat/Resources/config/suites/default.yml`). I think that some docs would be welcome, though in Sylius-Standard 1.4 it would be copied to the default config file also.